Resources/Textures/Objects/Weapons/Bombs/hot_potato.rsi/meta.json

```
@@ -1,23 +1,23 @@
 {
-  "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/1dbcf389b0ec6b2c51b002df5fef8dd1519f8068",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "icon"
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from https://github.com/vgstation-coders/vgstation13/commit/1dbcf389b0ec6b2c51b002df5fef8dd1519f8068, icon from potato.rsi & modified activated by Prole0 (GitHub)",
```

```suggestion
    "copyright": "icon from potato.rsi & modified activated by Prole0 (GitHub)",
```
since we replaced the old one we can remove the old link as well